Main.cpp

```diff
@@ -56,12 +56,12 @@ namespace RTE {
 	/// </summary>
 	void InitializeManagers() {
 		g_SettingsMan.Initialize();
+		g_WindowMan.Initialize();
 
 		g_LuaMan.Initialize();
 		g_NetworkServer.Initialize();
 		g_NetworkClient.Initialize();
 		g_TimerMan.Initialize();
-		g_WindowMan.Initialize();
```

Comment:
right now this _should_ still be compatible with multiplayer, since the only relevant change to that is post effects in gpu which were already client side.